packages/client/src/generation/TSClient/PrismaClient.ts

```diff
@@ -100,6 +100,29 @@ function executeRawDefinition(this: PrismaClientClass) {
   $executeRawUnsafe<T = unknown>(query: string, ...values: any[]): PrismaPromise<number>;`
 }
 
+function runCommandRawDefinition(this: PrismaClientClass) {
+  // we do not generate `$executeRaw...` definitions if not supported
+  if (!this.dmmf.mappings.otherOperations.write.includes('runCommandRaw')) {
+    return '' // https://github.com/prisma/prisma/issues/8189
+  }
+
+  return `
+  /**
+   * Executes a raw mongo command and returns the result of it.
+   * @example
+   * \`\`\`
+   * const user = await (prisma as any).$runCommandRaw({
+   *   aggregate: 'User',
+   *   pipeline: [{ $match: { name: 'Bob' } }, { $project: { email: true, _id: false } }],
+   *   explain: false,
+   * })
+   * \`\`\`
+   * 
+   * Read more in our [docs](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/raw-database-access).
+   */
+  $runCommandRaw(command: Prisma.InputJsonValue): PrismaPromise<Prisma.JsonObject>;`
```

Comment:
I guess `Prisma.InputJsonValue` doesn't help you much creating a raw command here, and we can't do more than that?

Reply:
The decision came from the [design doc](https://www.notion.so/prismaio/MongoDB-Query-Raw-a01774807b644b7894c10ecd53060718#3d73000c2d96486380b6dbc9a4bdad50), types change between mongo versions. It aligns with the rest of the raw APIs that are untyped too.